### PR TITLE
add eip155-61916.json DoKEN Mainnet

### PR DESCRIPTION
**Pull request by indowebdeveloper**
Adding DoKEN Mainnet including the logo

### CODE REVIEW

**Comment by indowebdeveloper** [comment]
any update regarding this ?

**Comment by indowebdeveloper** [comment]
Any Updates ? :D

**Review by ligi — CHANGES_REQUESTED** [review]
where did you pin the IPFS CID? I could not resolve it via `ipfs get`

**Comment by indowebdeveloper** [comment]
> where did you pin the IPFS CID? I could not resolve it via `ipfs get`

Can you update the CID to bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu ?

File: _data/icons/doken.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu",
+    "width": 1024,
+    "height": 1024,
+    "format": "png"
```
Comment (ligi) [inline _data/icons/doken.json:6]: identify bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu
bafkreifms4eio6v56oyeemnnu5luq3sc44hptan225lr45itgzu3u372iu PNG 200x200 200x200+0+0 8-bit sRGB 28797B 0.000u 0:00.000

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/doken.json (hunk 1 above)
Comment (indowebdeveloper) [reply _data/icons/doken.json:6]: Should i resize the image or just change the width and height in json as the corresponding image size?

Comment (indowebdeveloper) [reply _data/icons/doken.json]: I just updated the json to 200 x 200

**Review by ligi — APPROVED** [review]

**Comment by indowebdeveloper** [comment]
@ligi All good ? :D

**Pull request merged**